app/src/main/java/org/eyeseetea/malariacare/data/remote/PushDhisSDKDataSource.java

```diff
@@ -91,6 +102,44 @@ public void call(Throwable throwable) {
                 });
     }
 
+
+    private Map<String, PushReport> convertImportSummaryToPushReport(
```

Comment:
@idelcano Can you extract this method to mapper class and create a unit tests for verify map or conversion is valid?